frontend/src/widgets/StandardGoalList.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withWidgetData from './withWidgetData';
+import Container from '../components/Container';
+import WidgetH2 from '../components/WidgetH2';
+import SimpleSortableTable from '../components/SimpleSortableTable';
+import './StandardGoalList.css';
+
+export function StandardGoalsListTable({
+  data, loading, title,
+}) {
+  // Define the columns for the SimpleSortableTable
+  const columns = [
+    { key: 'name', name: 'Goal category' },
+    { key: 'count', name: '# of goals' },
+  ];
+
+  return (
+    <Container paddingX={1} paddingY={1} loading={loading} loadingLabel="Goals list loading">
```

Comment:
These props create extra padding that isn't in the mockup

Reply:
These props pre existed x needs to be 0 and y 1.